cli/internal/helm/BUILD.bazel

```diff
@@ -0,0 +1,373 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("//bazel/go:go_test.bzl", "go_test")
+
+go_library(
+    name = "helm",
+    srcs = [
+        "backup.go",
+        "client.go",
+        "loader.go",
+        "values.go",
+    ],
+    embedsrcs = [
+        "charts/cert-manager/Chart.yaml",
+        "charts/cert-manager/templates/NOTES.txt",
+        "charts/cert-manager/templates/_helpers.tpl",
+        "charts/cert-manager/templates/cainjector-deployment.yaml",
+        "charts/cert-manager/templates/cainjector-psp-clusterrole.yaml",
+        "charts/cert-manager/templates/cainjector-psp-clusterrolebinding.yaml",
+        "charts/cert-manager/templates/cainjector-psp.yaml",
+        "charts/cert-manager/templates/cainjector-rbac.yaml",
+        "charts/cert-manager/templates/cainjector-serviceaccount.yaml",
+        "charts/cert-manager/templates/crds.yaml",
+        "charts/cert-manager/templates/deployment.yaml",
+        "charts/cert-manager/templates/networkpolicy-egress.yaml",
+        "charts/cert-manager/templates/networkpolicy-webhooks.yaml",
+        "charts/cert-manager/templates/psp-clusterrole.yaml",
+        "charts/cert-manager/templates/psp-clusterrolebinding.yaml",
+        "charts/cert-manager/templates/psp.yaml",
+        "charts/cert-manager/templates/rbac.yaml",
+        "charts/cert-manager/templates/service.yaml",
+        "charts/cert-manager/templates/serviceaccount.yaml",
+        "charts/cert-manager/templates/servicemonitor.yaml",
+        "charts/cert-manager/templates/startupapicheck-job.yaml",
+        "charts/cert-manager/templates/startupapicheck-psp-clusterrole.yaml",
+        "charts/cert-manager/templates/startupapicheck-psp-clusterrolebinding.yaml",
+        "charts/cert-manager/templates/startupapicheck-psp.yaml",
+        "charts/cert-manager/templates/startupapicheck-rbac.yaml",
+        "charts/cert-manager/templates/startupapicheck-serviceaccount.yaml",
+        "charts/cert-manager/templates/webhook-config.yaml",
+        "charts/cert-manager/templates/webhook-deployment.yaml",
+        "charts/cert-manager/templates/webhook-mutating-webhook.yaml",
+        "charts/cert-manager/templates/webhook-psp-clusterrole.yaml",
+        "charts/cert-manager/templates/webhook-psp-clusterrolebinding.yaml",
+        "charts/cert-manager/templates/webhook-psp.yaml",
+        "charts/cert-manager/templates/webhook-rbac.yaml",
+        "charts/cert-manager/templates/webhook-service.yaml",
+        "charts/cert-manager/templates/webhook-serviceaccount.yaml",
+        "charts/cert-manager/templates/webhook-validating-webhook.yaml",
+        "charts/cert-manager/values.yaml",
+        "charts/cilium/.helmignore",
+        "charts/cilium/Chart.yaml",
+        "charts/cilium/LICENSE",
+        "charts/cilium/README.md",
```

Comment:
This very likely was already the case before, but we embed some redundant files which are not user-visible or used by our code.

Reply:
Yeah this was already the case and should not be solved here